### PR TITLE
Fixes and improvements to the Interactive Map

### PR DESCRIPTION
**Pull request by hcarneiro**
Issues references:
- https://github.com/Fliplet/fliplet-studio/issues/3824
- https://github.com/Fliplet/fliplet-studio/issues/3821
- https://github.com/Fliplet/fliplet-studio/issues/3822
- https://github.com/Fliplet/fliplet-studio/issues/3820
- https://github.com/Fliplet/fliplet-studio/issues/3825

This PR is aimed to fix all the listed above as well as the errors triggered on Sentry:
- https://sentry.io/fliplet/staging-widgets-interface/issues/878509295/?environment=staging
- https://sentry.io/fliplet/staging-widgets-interface/issues/878292785/?environment=staging
- https://sentry.io/fliplet/staging-widgets-interface/issues/877090145/?environment=staging

### CODE REVIEW

File: build.html [hunk 1]
```diff
@@ -1,80 +1,80 @@
-<div class="is-loading" data-interactive-floorplan-id="{{ id }}">
-  <div v-if="containsData" class="floorplan-component-wrapper">
-    <div class="floor-wrapper-holder">
-      <div class="floor-wrapper" v-for="(floor, index) in floors" v-bind:class="{ 'active': selectedFloorData && floor.id === selectedFloorData.id && imageLoaded }">
-        <div class="floor-container" :id="'floor-' + floor.id">
-          <img v-if="floor.image" class="floor-image" :src="floor.image.url"/>
+<div class="is-loading" data-interactive-map-id="{{ id }}">
```
Comment (tonytlwu) [inline build.html]: How about `{{ uuid }}`?

```suggestion
<div class="is-loading" data-interactive-map-id="{{ id }}" data-interactive-map-uuid="{{ uuid }}">
```

File: vendor/bootstrap-colorpicker.css [hunk 2]
```diff
@@ -1,5 +1,14 @@
 /*!
- * Bootstrap Colorpicker v2.5.2
+ * Bootstrap Colorpicker v2.5.1
```
Comment (tonytlwu) [inline vendor/bootstrap-colorpicker.css]: A downgrade? 🤔

Comment (hcarneiro) [reply vendor/bootstrap-colorpicker.css]: Yes. They changed how they show the colorpicker UI and I couldn't be bothered trying to make it work.
So I reverted to the same version we use the Appearance tab.

File: css/interface.css [hunk 3]
```diff
@@ -138,28 +134,37 @@ header .title-holder p {
...
 
 .panel-heading {
   position: relative;
   padding: 0;
 }
 
-.panel-heading:hover {
-  cursor: -webkit-grab;
-  cursor: -moz-grab;
+.grabbable {
```
Comment (tonytlwu) [inline css/interface.css]: Well, you could've stuck with `.panel-heading`, but I appreciate the reusable `.grabbale` class ☺️

File: js/libs/build.js [hunk 4]
```diff
@@ -83,35 +86,35 @@ Fliplet().then(function () {
...
+          this.pzElement = $('#map-' + this.selectedMapData.id)
 
           this.flPanZoomInstance = Fliplet.UI.PanZoom.create(this.pzElement, {
             maxZoom: 4,
             zoomStep: 0.25,
             animDuration: 0.1
           })
 
-          this.flPanZoomInstance.on('floorImageLoaded', (e) => {
+          this.flPanZoomInstance.on('mapImageLoaded', (e) => {
```
Comment (tonytlwu) [inline js/libs/build.js]: `e` isn't used

File: css/interface.css [hunk 5]
```diff
@@ -138,28 +134,37 @@ header .title-holder p {
...
+    cursor: -moz-grabbing;
+    cursor: -webkit-grabbing;
 }
 
 .panel-heading h4 {
   padding: 10px 40px 10px 6px;
 }
 
-.floor-markers-holder .panel-heading:hover {
+.map-markers-holder .panel-heading:hover {
```
Comment (tonytlwu) [inline css/interface.css]: You don't need `:hover` for cursors

**Pull request merged**